NAMESPACE

```diff
@@ -1,6 +1,17 @@
 # Generated by roxygen2: do not edit by hand
 
+export(detectStandard)
 export(eDISH)
 export(eDISHOutput)
+export(generateSettings)
+export(getRequiredColumns)
```

Comment:
What is the rationale behind exporting the "getXXX" functions - will the R user need these outside of detectStandard/generateSettings/validateSettings ?